pynestml/frontend/pynestml_frontend.py

```diff
@@ -313,7 +315,16 @@ def process():
 
     # perform build
     if not errors_occurred and _builder is not None:
-        _builder.build()
+        target_path = FrontendConfiguration.get_target_path()
+        stdout_path = os.path.join(target_path, "cmake_output.txt")
```

Comment:
I extended the `frontend_configuration` and `pynestml_frontend` with a new argument to support redirecting CMake output. The default behavior will be simply printing out the CMake output in the terminal, otherwise when the flag (argument) is provided, the CMake output will be redirected to a file.